src/Kovan-DssSpell.t.sol

```diff
@@ -1354,6 +1321,9 @@ contract DssSpellTest is DSTest, DSMath {
                     (exists, value) = clip.calc().call(abi.encodeWithSignature("cut()"));
```

Comment:
Maybe should also assert here that if there is a `step`, it is also non-zero.

Reply:
Done